packages/api/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { AsyncStorage } from "./AsyncStorage";
+export { StorageExtension } from "./StorageExtension";
+export * from "./StorageModel";
```

Comment:
Prefer explicit exports as it will aid tree shaking, and in this particular case, help bundlers understand that these are only types:
```suggestion
export type { StorageExtension } from "./StorageExtension";
export type { StorageKeys, StorageModel } from "./StorageModel";
```

Reply:
make total sense 🙏